### PR TITLE
Support week 7 episode

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/icraf_s01e07.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        "CodeID": "code-NIC-99631cb8",
+        "CodeType": "Control",
+        "ControlCode": "NIC",
+        "DisplayText": "NIC (not internally consistent)",
+        "NumericValue": -50,
+        "StringValue": "NIC",
+        "VisibleInCoda": false
+      },
+      {
+            "StringValue": "Noise_Other_Project",
```
Comment (as2388) [inline code_schemes/icraf_s01e07.json]:
```suggestion
        "StringValue": "Noise_Other_Project",
```

File: run_scripts/3_generate_outputs.sh [hunk 2]
```diff
@@ -37,7 +37,8 @@ cd ..
     "$DATA_ROOT/Raw Data/icraf_s01_e01_activation.json" "$DATA_ROOT/Raw Data/icraf_s01_e02_activation.json" \
     "$DATA_ROOT/Raw Data/icraf_s01_e03_activation.json" "$DATA_ROOT/Raw Data/icraf_s01_e04_activation.json" \
     "$DATA_ROOT/Raw Data/icraf_s01_e05_activation.json" "$DATA_ROOT/Raw Data/icraf_s01_e06_activation.json" \
-    "$DATA_ROOT/Raw Data/icraf_s01_demogs.json" "$DATA_ROOT/Raw Data/icraf_s01_follow_up_survey.json" \
+    "$DATA_ROOT/Raw Data/icraf_s01_e07_activation.json" "$DATA_ROOT/Raw Data/icraf_s01_demogs.json" \
+     "$DATA_ROOT/Raw Data/icraf_s01_follow_up_survey.json" \
```
Comment (as2388) [inline run_scripts/3_generate_outputs.sh]:
```suggestion
    "$DATA_ROOT/Raw Data/icraf_s01_follow_up_survey.json" \
```

File: src/lib/channels.py [hunk 3]
```diff
@@ -24,7 +25,9 @@ class Channels(object):
         ("2019-04-23T00:00:00+0300", "2019-04-26T07:00:00+0300"),
         ("2019-04-30T00:00:00+0300", "2019-05-03T07:00:00+0300"),
         ("2019-05-07T00:00:00+0300", "2019-05-10T07:00:00+0300"),
-        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300")
+        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300"),
+        ("2019-05-21T00:00:00+0300", "2019-05-23T07:00:00+0300")
+
```
Comment (as2388) [inline src/lib/channels.py]:
```suggestion
```

File: src/lib/channels.py [hunk 4]
```diff
@@ -33,7 +36,9 @@ class Channels(object):
         ("2019-04-26T07:00:00+0300", "2019-04-29T24:00:00+0300"),
         ("2019-05-03T07:00:00+0300", "2019-05-06T24:00:00+0300"),
         ("2019-05-10T07:00:00+0300", "2019-05-13T24:00:00+0300"),
-        ("2019-05-17T07:00:00+0300", "2019-05-20T24:00:00+0300")
+        ("2019-05-17T07:00:00+0300", "2019-05-20T24:00:00+0300"),
+        ("2019-05-24T07:00:00+0300", "2019-05-27T24:00:00+0300")
+
```
Comment (as2388) [inline src/lib/channels.py]:
```suggestion
```

File: src/lib/channels.py [hunk 5]
```diff
@@ -24,7 +25,9 @@ class Channels(object):
         ("2019-04-23T00:00:00+0300", "2019-04-26T07:00:00+0300"),
         ("2019-04-30T00:00:00+0300", "2019-05-03T07:00:00+0300"),
         ("2019-05-07T00:00:00+0300", "2019-05-10T07:00:00+0300"),
-        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300")
+        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300"),
+        ("2019-05-21T00:00:00+0300", "2019-05-23T07:00:00+0300")
```
Comment (as2388) [inline src/lib/channels.py]: Are these dates definitely correct? Seems to be one day shorter than usual

File: src/lib/channels.py [hunk 6]
```diff
@@ -72,6 +81,8 @@ class Channels(object):
         S01E04_KEY: S01E04_RANGES,
         S01E05_KEY: S01E05_RANGES,
         S01E06_KEY: S01E06_RANGES,
+        S01E07_KEY: S01E07_RANGES
+
```
Comment (as2388) [inline src/lib/channels.py]:
```suggestion
```

File: src/lib/pipeline_configuration.py [hunk 7]
```diff
@@ -17,6 +17,8 @@ class CodeSchemes(object):
     ICRAF_S01E04 = _open_scheme("icraf_s01e04.json")
     ICRAF_S01E05 = _open_scheme("icraf_s01e05.json")
     ICRAF_S01E06 = _open_scheme("icraf_s01e06.json")
+    ICRAF_S01E07 = _open_scheme("icraf_s01e07.json")
+
```
Comment (as2388) [inline src/lib/pipeline_configuration.py:21]:
```suggestion
```

File: src/translate_rapid_pro_keys.py [hunk 8]
```diff
@@ -12,7 +12,9 @@ class TranslateRapidProKeys(object):
         "Rqa_S01_E03 (Value) - icraf_s01_e03_activation": 3,
         "Rqa_S01_E04 (Value) - icraf_s01_e04_activation": 4,
         "Rqa_S01_E05 (Value) - icraf_s01_e05_activation": 5,
-        "Rqa_S01_E06 (Value) - icraf_s01_e06_activation": 6
+        "Rqa_S01_E06 (Value) - icraf_s01_e06_activation": 6,
+        "Rqa_S01_E07 (Value) - icraf_s01_e07_activation": 7
+
```
Comment (as2388) [inline src/translate_rapid_pro_keys.py]:
```suggestion
```

File: src/translate_rapid_pro_keys.py [hunk 9]
```diff
@@ -21,7 +23,9 @@ class TranslateRapidProKeys(object):
         3: "rqa_s01e03_raw",
         4: "rqa_s01e04_raw",
         5: "rqa_s01e05_raw",
-        6: "rqa_s01e06_raw"
+        6: "rqa_s01e06_raw",
+        7: "rqa_s01e07_raw"
+
```
Comment (as2388) [inline src/translate_rapid_pro_keys.py]:
```suggestion
```

**Review by as2388 — APPROVED** [review]

File: src/lib/channels.py (hunk 5 above)
Comment (IsaackMwenda) [reply src/lib/channels.py]: That was an error! Thank you for noticing

**Pull request merged**